listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -448,14 +439,22 @@ export default class BrainzPlayer extends React.Component<
       direction,
       progressMs,
       durationMs,
+      isActivated,
     } = this.state;
     const { APIService, youtubeAuth, spotifyAuth } = this.context;
+    // Determine if the user is authenticated to search & play tracks with any of the datasources
+    const hasDatasourceToSearch =
+      this.dataSources.findIndex((ds) =>
+        ds.current?.canSearchAndPlayTracks()
+      ) !== -1;
```

Comment:
This (and the associated message line 468) might or might not be useful depending on wether we have a default player (like youtube using API keys) or if they all require auth. It does appear for a flash we the page loads, but suppose it doesn't hurt much to have clear indications in any case.